scripts/modules/apm_agents.py

```diff
@@ -48,11 +57,7 @@ def _content(self):
             image=None,
             labels=None,
             logging=None,
-            environment={
```

Comment:
This is required to be able to update the environments correctly. I did try with different approaches but there were certain issues with the environment type and how it's internally manipulated for other services.